packages/edit-site/src/components/sidebar-edit-mode/navigation-menu-sidebar/navigation-inspector.js

```diff
@@ -168,7 +171,7 @@ export default function NavigationInspector() {
 	}, [ isLoadingInnerBlocks, hasLoadedInnerBlocks ] );
 
 	return (
-		<div className="edit-site-navigation-inspector">
+		<VStack>
```

Comment:
Another good find! I see this now when looking at the other conditions, like when there are no navigation menus and they are also empty (a condition that doesn't make sense to me).

But I definitely wasn't thorough enough in my tests, so I'll make note of this for future testing. 

To see the placeholders, did you modify the code? Or is there a way you could get them to appear on the front end? I haven't been able to match the condition without editing the code. At first I wasn't going to ask because it didn't seem important enough, but maybe there is a trick I'm missing so figured I'd ask. 🙂

Reply:
Yes, I just modified the code 😄 Personally I prefer Storybook-based development because UI states can be tested much more easily, but most of the Gutenberg repo does not have Storybook, so 🤷